content/en/real_user_monitoring/_index.md

```diff
@@ -126,8 +126,26 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
 
 Access RUM by navigating to [**Digital Experience > Performance Summary**][1].
 
+Select an application from the top navigation, or follow the setup instructions for [browser][15] or [mobile][16] to add your first application.
+
+{{< img src="real_user_monitoring/rum-performance-application-selector.png" alt="Select a RUM application" >}}
+
 **Tip**: To open RUM from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `real user monitoring`.
 
+## Performance monitoring summary
+
+{{< img src="real_user_monitoring/rum-performance-browser.png" alt="RUM Performance Monitoring summary page for a browser application" >}}
+
+The [RUM Performance Monitoring summary][1] page provides relevant and actionable insights for both web and mobile applications. You have a tailored experience for each platform that helps you:
+
+- **Focus on key metrics** by platform, such as the UI latency for web or mobile crashes
```

Comment:
```suggestion
- **Focus on key data points** by platform, such as the UI latency for web or mobile crashes
```

Those are currently event based. They will be metric based in the future with RUM Tail-Based, but I'd rather use a generic term